### DISCUSSION TITLE
iOS SDK: Possible to play in background, e.g. picture-in-picture?

### DISCUSSION THREAD

**Discussion by henrik** [ant-media/Ant-Media-Server#5352] [Q&A]
Hi,

Have not yet experimented with https://github.com/ant-media/WebRTC-iOS-SDK but considering it.

Could you tell me if it would make it possible to play audio or even video (picture-in-picture) when my app is backgrounded?

I ask because we currently [stream video inside a WKWebView](https://stackoverflow.com/questions/76653814/can-i-keep-webrtc-audio-or-video-from-wkwebview-playing-when-app-is-backgrounded) and my impression is that this does not support playing audio/video in background. So it would be good to know if switching to this SDK would make it possible.

**Comment by mekya** [comment]
Hi Henrik,
Thank you very much for your question. 

We've not worked picture-in-picture and background stuff to give instructions yet.
 
Fortunately, I think it's something that can be done in the application level. 

Moreover, the source code is open and if you want to have contribution, you're more than welcome :)

Cheers
Oz

**Reply by henrik** [reply]
Thank you! I wasn't sure because I assume picture-in-picture consists of handing over some video stream to the system, and it doesn't seem to support WebRTC natively. But it's good to know that you think it may be possible. It's not something I will look further into now, but maybe in future.

**Reply by mekya** [reply]
Thank you @henrik ,

Yes it's possible. Please let us know if we can help you proceed
